app/uk/gov/hmrc/customs/rosmfrontend/services/subscription/TaxEnrolmentsService.scala

```diff
@@ -30,6 +30,7 @@ class TaxEnrolmentsService @Inject()(taxEnrolmentsConnector: TaxEnrolmentsConnec
 
   private val serviceName = "HMRC-CUS-ORG"
 
+  // TODO This method is not necessary, we can remove it
```

Comment:
Can this call be used to verify that the enrolment succeeded?

Reply:
I think so, not this version of the method, because this checks for CDS, but we can add parameter (serviceName)

but for this I think it's better to use a new method to check this (with similar name and body), because if this logic is not necessary right now, we should remove all usage of this method and logic around it. The codebase will be smaller :)